demisto_sdk/commands/lint/linter.py

```diff
@@ -138,17 +138,20 @@ def run_dev_packages(self, no_flake8: bool, no_bandit: bool, no_mypy: bool, no_p
                                     lint_files=self._facts["lint_files"],
                                     modules=modules,
                                     pack_type=self._pkg_lint_status["pack_type"]):
-                # Run lint check on host - flake8, bandit, mypy
+                # Run lint check on host - flake8, bandit
+                # in python2 files mypy will run in local os and in python3 mypy will run in docker container
+                py_version = self._facts["python_version"]
                 if self._pkg_lint_status["pack_type"] == TYPE_PYTHON:
                     self._run_lint_in_host(no_flake8=no_flake8,
                                            no_bandit=no_bandit,
-                                           no_mypy=no_mypy,
+                                           no_mypy=no_mypy or 3 <= py_version,
```

Comment:
```suggestion
                                           no_mypy=no_mypy or py_version >= 3,
```
Easier to read, not a real issue.